cinch/site.yml

```diff
@@ -29,6 +29,7 @@
   hosts: jenkins_master
   roles:
     - jenkins_master
+    - ntp
```

Comment:
Fair point, if someone needs NTP on a Jenkins slave they can always include it.  I like the "minimal" concept.